doc/decoding/decoding_intro.rst

```diff
@@ -113,225 +106,190 @@ Loading the data into nilearn
----------------------
+.. warning::
 
-An estimator object
-...................
+    **Do not predict on data used by the fit: this would yield misleadingly
+    optimistic scores.**
 
-To perform decoding we need to use an estimator from the `scikit-learn
-<http://scikit-learn.org>` machine-learning library. This object can
-predict a condition label **y** given a set **X** of imaging data.
 
-A simple and yet performant choice is the `Support Vector Classifier
+A first estimator
+-----------------
+
+To perform decoding, we need a model that can learn some relations
+between **X** (the imaging data) and **y** the condition label. As a default,
+Nilearn uses `Support Vector Classifier
 <http://scikit-learn.org/stable/modules/svm.html>`_ (or SVC) with a
-linear kernel. The corresponding class, :class:`sklearn.svm.SVC`, needs
-to be imported from the scikit-learn.
-
-Note that the documentation of the object details all parameters. In
-IPython, it can be displayed as follows::
-
-    In [10]: svc?
-    Type:             SVC
-    Base Class:       <class 'sklearn.svm.libsvm.SVC'>
-    String Form:
-    SVC(kernel=linear, C=1.0, probability=False, degree=3, coef0=0.0, tol=0.001,
-    cache_size=200, shrinking=True, gamma=0.0)
-    Namespace:        Interactive
-    Docstring:
-        C-Support Vector Classification.
-        Parameters
-        ----------
-        C : float, optional (default=1.0)
-            penalty parameter C of the error term.
-    ...
+linear kernel. This is a simple yet performant choice that works in a wide
+variety of problems.
 
 .. seealso::
+  the `scikit-learn documentation on SVMs
+  <http://scikit-learn.org/stable/modules/svm.html>`_
 
-   the `scikit-learn documentation on SVMs
-   <http://scikit-learn.org/stable/modules/svm.html>`_
 
+Decoding made easy
+-------------------
 
-Applying it to data: fit (train) and predict (test)
-...................................................
 
-The prediction objects have two important methods:
+Nilearn makes it easy to train a model with a principled pipeline using the
+:class:`nilearn.decoding.Decoder` object. Using the mask we defined before
+and an SVC estimator as we already introduced, we can create a pipeline in
+two lines. The additional `standardize`=True argument adds a normalization
+of images signal to a zero mean and unit variance, which will improve
+performance of most estimators.
 
-- a `fit` function that "learns" the parameters of the model from the data.
-  Thus, we need to give some training data to `fit`.
-- a `predict` function that "predicts" a target from new data.
-  Here, we just have to give the new set of images (as the target should be
-  unknown):
+    >>> from nilearn.decoding import Decoder # doctest: +SKIP
+    >>> decoder = Decoder(estimator='svc', mask=mask_filename) # doctest: +SKIP
 
-.. warning::
+Then we can fit it on the images and the conditions we chose before.
+
+    >>> decoder.fit(fmri_niimgs, conditions) # doctest: +SKIP
 
```

Comment:
According to the above comment, shouldn't we replace that with a reference to an example ?

Reply:
I went with leaving two or three really basic lines that don't risk to be deprecated in the coming years for pedagogy. But really not much